src/_includes/footer.html

```diff
@@ -1,18 +1,48 @@
+          <img src="{{'/images/beam_logo_circle.svg'|prepend:site.baseUrl}}" class="footer__logo" alt="Beam logo">
+        </div>
+        <div class="footer__cols__col__logo">
+          <img src="{{'/images/apache_logo_circle.svg'|prepend:site.baseUrl}}" class="footer__logo" alt="Apache logo">
+        </div>
       </div>
+      <div class="footer__cols__col footer__cols__col--md">
+        <div class="footer__cols__col__title">Start</div>
+        <div class="footer__cols__col__link"><a href="{{'/get-started/beam-overview/'|prepend:site.baseUrl}}">Overview</a></div>
+        <div class="footer__cols__col__link"><a href="{{'/get-started/quickstart-java/'|prepend:site.baseUrl}}">Quickstart (Java)</a></div>
+        <div class="footer__cols__col__link"><a href="{{'/get-started/quickstart-python/'|prepend:site.baseUrl}}">Quickstart (Python)</a></div>
+        <div class="footer__cols__col__link"><a href="{{'/get-started/downloads/'|prepend:site.baseUrl}}">Downloads</a></div>
+      </div>
+      <div class="footer__cols__col footer__cols__col--md">
+        <div class="footer__cols__col__title">Docs</div>
+        <div class="footer__cols__col__link"><a href="{{'/documentation/programming-guide/'|prepend:site.baseUrl}}">Concepts</a></div>
+        <div class="footer__cols__col__link"><a href="{{'/documentation/pipelines/design-your-pipeline/'|prepend:site.baseUrl}}">Pipelines</a></div>
+        <div class="footer__cols__col__link"><a href="{{'/documentation/runners/capability-matrix/'|prepend:site.baseUrl}}">Runners</a></div>
+      </div>
+      <div class="footer__cols__col footer__cols__col--md">
+        <div class="footer__cols__col__title">Community</div>
+        <div class="footer__cols__col__link"><a href="{{'/contribute/'|prepend:site.baseUrl}}">Contribute</a></div>
+        <div class="footer__cols__col__link"><a href="{{'/contribute/team/'|prepend:site.baseUrl}}">Team</a></div>
+        <div class="footer__cols__col__link"><a href="{{'/contribute/presentation-materials/'|prepend:site.baseUrl}}">Media</a></div>
+      </div>
+      <div class="footer__cols__col footer__cols__col--md">
+        <div class="footer__cols__col__title">Resources</div>
+        <div class="footer__cols__col__link"><a href="{{'/blog/'|prepend:site.baseUrl}}">Blog</a></div>
+        <div class="footer__cols__col__link"><a href="{{'/get-started/support/'|prepend:site.baseUrl}}">Support</a></div>
+        <div class="footer__cols__col__link"><a href="https://github.com/apache/beam">GitHub</a></div>
+      </div>
+    </div>
+  </div>
+  <div class="footer__bottom">
+    &copy;
+    <a href="http://www.apache.org">The Apache Software Foundation</a>
+    | <a href="{{ site.baseurl }}/privacy_policy">Privacy Policy</a>
+    | <a href="{{ "/feed.xml" | prepend: site.baseurl }}">RSS Feed</a>
+    <br><br>
+    Apache Beam, Apache, the Beam logo, and the Apache feather logo are either
+    registered trademarks or trademarks of The Apache Software Foundation.
```

Comment:
Add "All other products or name brands are trademarks of their respective holders, including The Apache Software Foundation."